radiation/python/radlw/radsw_param.py

```diff
@@ -18,86 +16,20 @@
 ntbmx = 10000
 # SW bands counter starting index (for compatibility with previous
 # SW radiation schemes)
-NSWSTR = 1
-
-NSWEND = nbands
+nswstr = 1
 nbdsw = nbands
 
-# The actual number of g-point for bands 16-29
-NG16 = 6
-NG17 = 12
-NG18 = 8
-NG19 = 8
-NG20 = 10
-NG21 = 10
-NG22 = 2
-NG23 = 10
-NG24 = 8
-NG25 = 6
-NG26 = 6
-NG27 = 8
-NG28 = 6
-NG29 = 12
-
-NG = [
-    NG16,
-    NG17,
-    NG18,
-    NG19,
-    NG20,
-    NG21,
-    NG22,
-    NG23,
-    NG24,
-    NG25,
-    NG26,
-    NG27,
-    NG28,
-    NG29,
-]
-
-# Accumulative starting index for bands 16-29
-NS16 = 0
-NS17 = NS16 + NG16
-NS18 = NS17 + NG17
-NS19 = NS18 + NG18
-NS20 = NS19 + NG19
-NS21 = NS20 + NG20
-NS22 = NS21 + NG21
-NS23 = NS22 + NG22
-NS24 = NS23 + NG23
-NS25 = NS24 + NG24
-NS26 = NS25 + NG25
-NS27 = NS26 + NG26
-NS28 = NS27 + NG27
-NS29 = NS28 + NG28
+idxsfc = [1, 1, 1, 1, 1, 1, 1, 1, 0, 2, 2, 2, 2, 1]  # band index for sfc flux
+idxebc = [5, 5, 4, 4, 3, 3, 2, 2, 1, 1, 1, 1, 1, 5]  # band index for cld prop
 
-# array contains values of NS16-NS29
-NGS = [
```

Comment:
is this just folded into a different variable?

Reply:
Very odd, I'll look into it

Reply:
I renamed it to lower case ngs, the diff just made it look weird. I think I moved a lot of things around, but nothing got lost (no conflict with radiation_dev branch)